Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/AWSCredentialIdentityResolvers/SSOAWSCredentialIdentityResolver.swift

```diff
@@ -62,14 +79,18 @@ public struct SSOAWSCredentialIdentityResolver: AWSCredentialIdentityResolver {
 
         var ssoToken: BearerTokenIdentity!
         var region: String!
+        var tokenFeatureIDs: [String]!
         if let ssoSessionName {
             region = try getProperty(ssoSessionName, .ssoSession, "sso_region", fileBasedConfig)
+            tokenFeatureIDs = [CredentialFeatureID.CREDENTIALS_PROFILE_SSO.rawValue]
             ssoToken = try await SSOBearerTokenIdentityResolver(
                 profileName: resolvedProfileName,
-                configFilePath: configFilePath
+                configFilePath: configFilePath,
+                credentialFeatureIDs: credentialFeatureIDs + tokenFeatureIDs
             ).getIdentity(identityProperties: identityProperties)
         } else { // Handle Legacy token flow.
             region = try getProperty(resolvedProfileName, .profile, "sso_region", fileBasedConfig)
+            tokenFeatureIDs = [CredentialFeatureID.CREDENTIALS_PROFILE_SSO_LEGACY.rawValue]
```

Comment:
Tracks feature IDs for how SSO token was resolved from (legacy vs. `sso_token` section)